centreon/src/Core/Notification/Domain/Model/NotificationResource.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Notification\Domain\Model;
+
+use Centreon\Domain\Common\Assertion\AssertionException;
+
+class NotificationResource
+{
+    public const HOSTGROUP_RESOURCE_TYPE = 'hostgroup';
+    public const SERVICEGROUP_RESOURCE_TYPE = 'servicegroup';
```

Comment:
Not used constant
```suggestion
```

Reply:
This is probably the fact that something was not propery coded somewhere ? 🤔

Reply:
Don't know it was like this when i got the subject, will remove it.